spec/spec_helper.rb

```diff
@@ -12,18 +12,26 @@
   c.cassette_library_dir = 'spec/pivo_flow/vcr_cassettes'
 end
 
-def stub_git_config(options = {})
-  git_options = {
-    "pivo-flow.api-token" => "testtesttesttesttesttesttesttest",
-    "pivo-flow.project-id" => "123456",
-    "user.name" => "Adam Newman"
-  }.merge options
-  Grit::Repo.stub!(:new).and_return mock('Grit::Repo', :config => git_options)
+module StubHelpers
+  def stub_git_config(options = {})
+    git_options = {
+      "pivo-flow.api-token" => "testtesttesttesttesttesttesttest",
+      "pivo-flow.project-id" => "123456",
+      "user.name" => "Adam Newman"
+    }.merge(options)
 
+    allow(Grit::Repo).to receive(:new) do
+      instance_double('Grit::Repo', :config => git_options)
```

Comment:
can we use a newer hash syntax here? `key: value`

Reply:
Yeah definitely, I think I was just trying to not modify as much as possible and leave what was there. I can update it though.,